translator/setMetricPath.go

```diff
@@ -28,7 +28,9 @@ func SetMetricPath(result map[string]interface{}, sectionKey string) {
 				input := inputIntf.(map[string]interface{})
 				if val, ok := input[tagKey]; ok {
 					tags := val.(map[string]interface{})
-					tags[routingTagKey] = sectionKey
+					if _, ok := tags[routingTagKey]; !ok {
+						tags[routingTagKey] = sectionKey
+					}
```

Comment:
I am not fully understand this case. Could you explain more about why  routingTagKey will conflict.

Reply:
For new plugin to work, we have translator changes under logs which gets **logs** metricPath by default see below:
``` 
[inputs.myPlugin.tags]
    metricPath = "logs"
```
Since we are emitting metrics using **outputs.cloudwatchlogs**, we need tags metrics path to set as **metrics**.
```
[inputs.myPlugin.tags]
   metricPath = "metrics"
```

Reply:
Could you add annotation here?

Reply:
I added the comment.